### ISSUE TITLE
Integration with Socialite 

### ISSUE THREAD

**Issue by benmag** [ohmybrew/laravel-shopify#59]
Are there any plans to make the OAuth system in this package also work with the Laravel Socialite? I can give it a try and send a PR if it's wanted

**Comment by ohmybrew** [comment]
@benmag  Currently not on my plate, no. Would it even be possible? The underlying auth stuff comes from [my other package](https://github.com/ohmybrew/Basic-Shopify-API). It seems Socialite would override that?

**Comment by benmag** [comment]
Yeah, I think Socialite would likely override that or at least parts of that

**Comment by ohmybrew** [comment]
@benmag Know any good resources for integrating with Socialite? I'd like to see how feasible it would be.

**Comment by benmag** [comment]
I'd suggest checking out the community Socialite Providers (https://socialiteproviders.github.io/about.html) and then the Shopify Socialite provider: https://github.com/SocialiteProviders/Shopify

There might be a way to bake a Socialite provider into the package that can be enabled like any other Socialite provider
